core/src/main/resources/jenkins/model/Jenkins/configure.jelly

```diff
@@ -62,7 +62,7 @@ THE SOFTWARE.
 
       <f:bottomButtonBar>
         <f:submit value="${%Save}" />
-        <f:apply value="${%Apply}"/>
+        <f:apply />
```

Comment:
While the followup change to https://github.com/jenkinsci/jenkins/pull/3287 is not filed, this will remove existing translations without replacement.

Reply:
#3287 already been merged.

Reply:
Yes, without moving the existing localizations to the `f:apply` control. It's similar to the discussion in https://github.com/jenkinsci/jenkins/pull/3319.

If we merge this change, system configuration will lose the apply button localization for Bulgarian, German, French, Italian, etc., as there are no corresponding translations to https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/lib/form/apply.jelly -- only English and Chinese.

Reply:
It's my mistake. So your option is put them back, or remove control to `f:apply`.

Reply:
In my opinion, the preferable option --as in #3319 -- would be to just move the translations to `apply_*.properties` files.

If you don't wish to work on translations of languages you don't know, keeping the translations around and reverting part of this PR would also work.

Reply:
Ok than, I move those files.